scripts/golangci.yml

```diff
@@ -0,0 +1,152 @@
+---
+# This file contains all available configuration options
+# with their default values.
+
```

Comment:
Request you mention the source for this file and if any options are changed here? Helps change things in the future or catch up to later lint configuration changes as they happen.

Reply:
done